test/solidity_dynamic_strings.js

```diff
@@ -0,0 +1,54 @@
+const Web3 = require("web3");
+const assert = require("assert");
+const Ganache = require("../index");
+const path = require("path");
+const compileAndDeploy = require("./helpers/contracts").compileAndDeploy;
+
+const mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
```

Comment:
The mnemonic defines the list of accounts that ganache will use - but you're using web3.eth.getAccounts below (which is good) so there's no need to hard code a mnemonic.

It's better if tests use randomized accounts - we want tests to look like best practice code examples.

Reply:
Great suggestions for overall refactoring efforts.